os_net_config/common.py

```diff
@@ -95,16 +95,17 @@ def get_noop():
 
 
 def log_exceptions(type, value, tb):
-    logger.exception(''.join(traceback.format_exception(
-        type, value, tb)))
+    logger.exception("".join(traceback.format_exception(type, value, tb)))
     # calls default excepthook
     sys.__excepthook__(type, value, tb)
```

Comment:
This is horrible. logger.exception already deals with extracting the stack trace. Please remove this function.